nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -177,6 +177,26 @@ private static void reportMismatchedTypeForTernaryOperator(
             errorMessage, analysis.buildDescription(tree), state, null));
   }
 
+  private void reportInvalidParametersNullabilityError(
+      Type formalParameter,
+      Type actualParameter,
+      ExpressionTree expressionTree,
+      VisitorState state,
+      NullAway analysis) {
+    ErrorBuilder errorBuilder = analysis.getErrorBuilder();
+    ErrorMessage errorMessage =
+        new ErrorMessage(
+            ErrorMessage.MessageTypes.TYPE_PARAMETER_CANNOT_BE_NULLABLE,
```

Comment:
we'll want a new type of error here

Reply:
something like GNERIC_TYPE_PARAMETER_CANNOT_BE_NULLABLE ?

Reply:
Maybe `PASS_NULLABLE_GENERIC`